oeps/oep-0019-bp-developer-documentation.rst

```diff
@@ -115,11 +114,11 @@ ADRs
        new members, to understand discovered historical traces, to refer back to past decisions,
        and to support external oversight.
    * - **Location in GitHub**
-     - A `local decisions`_ folder will be created within a **docs** folder in the repo/app/folder. 
+     - A local `decisions`_ folder will be created within a **docs** folder in the repo/app/folder. 
```

Comment:
This is intended to be a link to an example **decisions** folder. See the link below to https://github.com/edx/credentials/tree/master/docs/decisions.

Since this got you, a rewording here would clarify for future readers as well.

Reply:
Done.